src/Simplex/Messaging/Protocol.hs

```diff
@@ -419,6 +419,7 @@ data Command (p :: Party) where
   NEW :: NewQueueReq -> Command Recipient
   SUB :: Command Recipient
   KEY :: SndPublicAuthKey -> Command Recipient
+  RKEY :: NonEmpty RcvPublicAuthKey -> Command Recipient
```

Comment:
rfc suggests RADD etc. to avoid race conditions?

Reply:
It won’t help - there is also RSET